03week/towersOfHanoi.js

```diff
@@ -48,17 +55,16 @@ return true;
 }
 }
 
-
-
+// when 4 values are in  order in the end stack which is our last array "c" the game has been won 
 const checkForWin=()=> {
-  if (stacks.c.length === 4){
+  if (stacks.c.length === 4 || stacks.b.length === 4){
```

Comment:
fix the indentation here and this function can be greatly simplified, you can just return the evaluation